cmd/version.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+
+	goVersion "github.com/christopherhein/go-version"
+	"github.com/spf13/cobra"
+)
+
+var (
+	shortened = false
+	version   string
+	commit    string
+	date      string
+)
+
+func versionCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Version will output the current build information",
+		Long:  ``,
+		Run: func(_ *cobra.Command, _ []string) {
+			var response string
+			versionOutput := goVersion.New(version, commit, date)
+
+			if shortened {
+				response = versionOutput.ToShortened()
+			} else {
+				response = versionOutput.ToJSON()
+			}
+			fmt.Printf("%+v", response)
```

Comment:
This should be re-written a touch for a few different reasons: a) we want to move the declaration of `response` as close to its usage site as possible, b) rather than relying on `fmt.Printf` to print to stdout for us, lets the form that accepts `io.Writer`.

```go
dest := os.Stdout

version := goVersion.New(version, commit, date)
var out string
if short {
	out = version.ToShortened()
} else {
	out = version.ToJSON()
}
_, err := fmt.Fprintf(dest, "%+v", out)
return err
```

note: part of me also likes re-writing it as:

```go
print := func(s string) error {
	_, err := fmt.Fprintf(os.Stdout, "%+v", s)
	return err
}

version := goVersion.New(version, commit, date)
if short {
	return print(version.ToShortened())
}
return print(version.ToJSON())
```

The first is likely more idiomatic, though ideally you want to remove the nesting. You could argue this is fairly idiomatic too:

```go
dest := os.Stdout

version := goVersion.New(version, commit, date)
if short {
	_, err := fmt.Fprint(os.Stdout, "%+v", version.ToShortened())
	return err
}
_, err := fmt.Fprintf(dest, "%+v", version.ToJSON())
return err
```